pony-sync.md

```diff
@@ -0,0 +1,35 @@
+# Pony Sync
+
+The Pony Sync is a weekly meeting where Pony contributors discuss topics related to Pony, including recently added or modified issues.
+
+## Running the Pony Sync
+
+The Pony Sync takes place as a [Zoom](https://zoom.us) call. In order to run the Pony Sync you must have Zoom installed, and be logged in as `ponylang.main@gmail.com`. A list of recently modified issues can be generated using the [pony-sync-helper]() application; this list should be pasted into the `sync` stream in Zulip, in a topic whose name is the date of the meeting in US date format (for example May 12, 2020 would be "5/12/2020").
+
+### Prerequisites
+
+In order to run the Pony Sync meeting you will need:
+
+1. A GitHub account
```

Comment:
I'm not a fan of markdown numbered list like this, it makes looking at the source weird.

Reply:
I like it because I don't have renumber the list if I add anything, but I'm not strongly committed to that so I can change it.

Reply:
let's see what others think. I agree with the renumbering. It's why I always use `-` instead.

Reply:
I made the change.